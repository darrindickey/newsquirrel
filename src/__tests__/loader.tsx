import 'jest-dom/extend-expect';
import * as React from 'react'
import {render} from 'react-testing-library';
import Loader from '../components/loader/loader';
import LoaderSize from '../enums/loaderSize';
test('Renders loader correctly', () => {
  const props = {
    size: LoaderSize.LARGE,
    start: true,
    stop: false,
  }
  const{getByTestId} = render(<Loader {...props}/>);
  
  const node = getByTestId('rt-loader');
  expect(node).toBeInTheDocument();
  expect(node).toMatchSnapshot();
})
