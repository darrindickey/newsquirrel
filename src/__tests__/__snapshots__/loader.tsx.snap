// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders loader correctly 1`] = `
<div
  class="sc-bdVaJa csFBTh"
  data-testid="rt-loader"
  size="1"
>
  <div
    class="sc-bwzfXH jhrhgn"
    size="1"
  />
</div>
`;
